haas/cli.py

```diff
@@ -98,16 +99,19 @@ def check_status_code(response):
 
 # TODO: This function's name is no longer very accurate.  As soon as it is
 # safe, we should change it to something more generic.
+
+
```

Comment:
Ug, we really shouldn't separate this from the code it's commenting below. Did the tool complain about this? Maybe put it inside the function if you have to.

Also, folks, that comment is ancient; can we add fixing that to someone's to do list?